### PR TITLE
fix: Incorrect duplicate filename handling when exporting files

### PR DESCRIPTION
**Pull request by dhruvanbhalara**
- fix: https://github.com/ReVanced/revanced-manager/issues/1527

### CODE REVIEW

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

File: pubspec.yaml [hunk 1]
```diff
@@ -79,6 +79,10 @@ dependencies:
   synchronized: ^3.1.0
   connectivity_plus: ^5.0.1
   flutter_file_dialog: ^3.0.2
+  cr_file_saver:
+    git:
+      url: https://github.com/dhruvanbhalara/cr_file_saver
+      ref: "fix/incorrect_file_name"
```
[the remark this answers is not in the document]
Comment (dhruvanbhalara) [reply pubspec.yaml:85]: Sure, will update with particular reference commit.

**Comment by dhruvanbhalara** [comment]
> Hey, this change didn't fix it for me
> image
> 

Can you please remove local cloned package once and try again?

**Comment by Ushie** [comment]
Locally building the APK then testing worked as intended
image

I think this may be an issue with our PR build workflow in this case

**Comment by TheAabedKhan** [comment]
Why are you guys doing all these??

This line fixes the issue.

image

**Pull request merged**